src/tokenizer.js

```diff
@@ -804,10 +804,12 @@ export default class Tokenizer {
     let hasEscape = this.index - start !== id.length;
 
     let type = this.getKeyword(id);
-    if (hasEscape && type !== TokenType.IDENTIFIER) {
-      type = TokenType.ESCAPED_KEYWORD;
+    if (hasEscape) {
+      if (type !== TokenType.IDENTIFIER) {
```

Comment:
You can `&&` this condition with the previous one to combine these nested `if`s.